desktop/components/auth_modal/view.coffee

```diff
@@ -105,6 +106,7 @@ module.exports = class AuthModalView extends ModalView
     @$('button').attr 'data-state', 'loading'
 
     @user.set (data = @serializeForm())
+    @user.set(signupIntent: @signupIntent)
```

Comment:
I think the field should be `@user.set(sign_up_intent: @signupIntent)` to match the API below.